source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py

```diff
@@ -1412,3 +1413,143 @@ def sample_cylinder(
     xyz[..., 2].uniform_(h_min, h_max)
     # return positions
     return xyz
+
+
+"""
+Orientation Conversions
+"""
+
+
+def convert_orientation_convention(
```

Comment:
Can we rename this to something meaninful? It is confusing now compared to `convert_quat` which handles "wxyz" and "xyzw".